config.ini

```diff
@@ -0,0 +1,19 @@
+[Default]
+PORT: 8084
+HOST: 127.0.0.1
```

Comment:
Can be removed (sdx_transform_cora_url will use different variables to be figured out)